src/util/scanAdapters/EtherscanV1ScanAdapter.ts

```diff
@@ -49,12 +49,36 @@ export function makeEtherscanV1ScanAdapter(
     }
     const data = await response.json()
 
-    // console.log(data)
```

Comment:
Oh, this should have been deleted in a previous commit. Either that, or just squash all the commits together.